src/test/scala/com/typesafe/scalalogging/LoggerWithTaggedArgsSpec.scala

```diff
@@ -27,16 +27,18 @@ class LoggerWithTaggedAargsSpec extends AnyWordSpec with Matchers with Varargs {
       val f = fixture(_.isErrorEnabled)
       import f._
       noException shouldBe thrownBy {
-        logger.error("This should not throw: {}, {}", arg1, arg2)
+        logger.error("This should not throw: {}, {} - {}", arg1, arg2, arg3)
       }
+      verify(underlying).error(any[String], *, *, *)
```

Comment:
Had to add the verification due to strict-mode (if a mock method is call that wasn't either stub nor verified the test will fail)
I had to add the 3rd param as the macro always calls the varargs version of the Slf4j class, but on the direct call to verify, the compiler would pick the overloaded non-vararg version and the verification would fail as they are effectively different methods. By using 3 arguments the problem goes away as the only option is the varargs method